<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureWindow</class>
 <widget class="QWidget" name="ConfigureWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>91</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources/biobloom.qrc">
     <normaloff>:/BackButtonIcon.png</normaloff>:/BackButtonIcon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>85</width>
     <height>110</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ApplyButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>91</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources/biobloom.qrc">
     <normaloff>:/ApplyButtonIcon.PNG</normaloff>:/ApplyButtonIcon.PNG</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>85</width>
     <height>110</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Resources/biobloom.qrc"/>
 </resources>
 <connections/>
</ui>
